src/plot.cpp

```diff
@@ -619,6 +638,38 @@ Plot::set_mask(pugi::xml_node plot_node)
   }
 }
 
+void Plot::set_overlap_color(pugi::xml_node plot_node) {
+  color_overlaps_ = false;
+  if (check_for_node(plot_node, "show_overlaps")) {
+    color_overlaps_ = get_node_value_bool(plot_node, "show_overlaps");
+    overlap_color_ = RED;
```

Comment:
Probably should set this as default in the class definition itself?

Reply:
Definitely. Good call.